extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -524,8 +545,8 @@ public static function getHostById( $cityId ) {
 		$hostName = \WikiFactory::getVarValueByName( 'wgServer', $cityId );
 
 		if ( !empty( $wgDevelEnvironment ) ) {
-			if ( strpos( $hostName, "wikia.com" ) ) {
-				$hostName = str_replace( "wikia.com", "{$wgDevelEnvironmentName}.wikia-dev.com", $hostName );
+			if ( strpos( $hostName, static::DOT_WIKIA_COM ) ) {
+				$hostName = str_replace( static::DOT_WIKIA_COM, "{$wgDevelEnvironmentName}.wikia-dev.com", $hostName );
```

Comment:
Hmm, wouldn't getLocalEnvUrl handle this transformation? There's a handler for dev env there too.